sdk/eventhub/azure-eventhubs/azure/eventhub/_eventprocessor/checkpoint_store.py

```diff
@@ -8,72 +8,75 @@
-                - etag
+                - `fully_qualified_namespace` (str): The fully qualified namespace that the Event Hub belongs to.
+                  The format is like "<namespace>.servicebus.windows.net"
+                - `eventhub_name` (str): The name of the specific Event Hub the checkpoint is associated with,
+                  relative to the Event Hubs namespace that contains it.
+                - `consumer_group` (str): The name of the consumer group the ownership are associated with.
+                - `partition_id` (str): The partition ID which the checkpoint is created for.
+                - `owner_id` (str): A UUID representing the owner attempting to claim this partition.
+                - `last_modified_time` (datetime.datetime): The last time this ownership was claimed.
+                - `etag` (str): The Etag value for the last time this ownership was modified. Optional depending
+                  on storage implementation.
         """
 
     @abstractmethod
     def update_checkpoint(self, checkpoint):
         # type: (Dict[str, Union[str, int]]) -> None
-        """
-        Updates the checkpoint using the given information for the associated partition and
+        """Updates the checkpoint using the given information for the offset, associated partition and
         consumer group in the chosen storage service.
 
         Note: If you plan to implement a custom checkpoint store with the intention of running between
         cross-language EventHubs SDKs, it is recommended to persist the offset value as an integer.
 
         :param Dict[str,Any] checkpoint: A dict containing checkpoint information:
 
-                - fully_qualified_namespace (str): The fully qualified namespace that the event hub belongs to.
+                - `fully_qualified_namespace` (str): The fully qualified namespace that the Event Hub belongs to.
                   The format is like "<namespace>.servicebus.windows.net"
-                - eventhub_name (str): The name of the specific Event Hub the checkpoint is associated with, relative to
-                  the Event Hubs namespace that contains it.
-                - consumer_group (str): The name of the consumer group the ownership are associated with.
-                - partition_id (str): The partition id which the checkpoint is created for.
-                - sequence_number (int): The sequence_number of the :class:`EventData<azure.eventhub.EventData>`
+                - `eventhub_name` (str): The name of the specific Event Hub the checkpoint is associated with,
+                  relative to the Event Hubs namespace that contains it.
+                - `consumer_group` (str): The name of the consumer group the checkpoint is associated with.
+                - `partition_id` (str): The partition ID which the checkpoint is created for.
+                - `sequence_number` (int): The sequence number of the :class:`EventData<azure.eventhub.EventData>`
```

Comment:
These values are provided by the partition context, so the user will get both regardless.
While it is _currently_ optional that they persist the sequence number in a custom CheckpointStore implementation, I'd rather not document it as such in case the behaviour changes later.